.github/workflows/layer-publish.yml

```diff
@@ -53,7 +53,7 @@ jobs:
             LAYER_NAME=$LAYER_NAME-${{ inputs.release-group }}
           fi
           
-          LAYER_VERSION=$(echo "$GITHUB_REF_NAME" | sed -r 's/.*\/[^\d\.]*//g')
+          LAYER_VERSION=$(echo "$GITHUB_REF_NAME" | sed -r 's/.*\/[^0-9\.]*//g')
```

Comment:
The previous regex was leaving out the first digit for some reason.

Reply:
Yeah, I added `\d` not realizing that sed doesn't recognize it.  Thanks for fixing.